### PR TITLE
admission registration

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1531439

/assign mfojtik

### CODE REVIEW

File: vendor/k8s.io/kubernetes/plugin/pkg/admission/eventratelimit/admission.go [hunk 1]
```diff
@@ -43,6 +44,10 @@ func Register(plugins *admission.Plugins) {
 			}
 			return newEventRateLimit(configuration, realClock{})
 		})
+
+	// add our config types
+	eventratelimitapi.AddToScheme(plugins.ConfigScheme)
```
Comment (mfojtik) [inline vendor/k8s.io/kubernetes/plugin/pkg/admission/eventratelimit/admission.go]: need imagepolicy as well?

Comment (deads2k) [reply vendor/k8s.io/kubernetes/plugin/pkg/admission/eventratelimit/admission.go]: > need imagepolicy as well?

didn't come up in kube.  probably.

**Comment by deads2k** [comment]
closing in favor of https://github.com/openshift/origin/pull/18111

**Pull request closed without merging**